### DISCUSSION TITLE
MIMIC-IV ED patients fluid volumes

### DISCUSSION THREAD

**Discussion by 030vegetable** [MIT-LCP/mimic-code#1638] [MIMIC-IV-ED]
I need help with a question. Thank you very much for your answer.
I want to figure out In which module can I find large fluid volumes (such as those used for resuscitation) of ED patients? 
Are these in the mimiciv_hosp.prescriptions?
But i find that the data collected in this table is inconsistent with the amount of fluids ED used on the discharge records. 
May I ask in which table the large bags of fluids for ED patients are stored?
I would appreciate it if you could respond to my questions

**Comment by alistairewj** [comment]
I think this data is very hard to collect as it is inconsistently documented. Large volumes are not prescriptions so IIRC they are not in prescriptions. @li-lcp did a project focusing on this and might know more.

**Reply by 030vegetable** [reply]
Thank you very much for your reply
